data/5e/wizards_of_the_coast/core/universal_rules/ur_abilities.lst

```diff
@@ -1,281 +1,281 @@
-#
-CATEGORY=Internal|Default.MOD																														BONUS:VAR|AC_Ability_Score|min(AC_Ability_Score_Bonus,AbilityScoreLimit)|TYPE=Ability
-
-CATEGORY=Internal|Default.MOD					DEFINE:AbilityScoreLimit|0																					BONUS:VAR|AbilityScoreLimit|min(MODEQUIPMAXDEX,MXDXEN)
-#min(MaxAbilityScore_Bonus__Encumbrance,MaxAbilityScore_Bonus__Equipment)
-
-CATEGORY=Internal|Default.MOD																														BONUS:VAR|AC_Ability_Score_Bonus|DEX|PREVAREQ:DisableDexForAC,0
-
-CATEGORY=Internal|Default.MOD													ABILITY:Internal|AUTOMATIC|Movement Mode
-Movement Mode			CATEGORY:Internal
-
-###Block
-# Ability Name				Define
-CATEGORY=Internal|Movement Mode.MOD	DEFINE:Move_Walk|0	DEFINE:Move_Swim|0	DEFINE:Move_Fly|0	DEFINE:Move_Climb|0	DEFINE:Move_Burrow|0
-CATEGORY=Internal|Movement Mode.MOD	DEFINE:RaceBase_Walk|0	DEFINE:RaceBase_Swim|0	DEFINE:RaceBase_Fly|0	DEFINE:RaceBase_Climb|0	DEFINE:RaceBase_Burrow|0
-###Block
-# Ability Name				Modify VAR
-CATEGORY=Internal|Movement Mode.MOD	BONUS:VAR|Move_Walk|RaceBase_Walk|TYPE=Base
-CATEGORY=Internal|Movement Mode.MOD	BONUS:VAR|Move_Swim|RaceBase_Swim|TYPE=Base
-CATEGORY=Internal|Movement Mode.MOD	BONUS:VAR|Move_Fly|RaceBase_Fly|TYPE=Base
-CATEGORY=Internal|Movement Mode.MOD	BONUS:VAR|Move_Climb|RaceBase_Climb|TYPE=Base
-CATEGORY=Internal|Movement Mode.MOD	BONUS:VAR|Move_Burrow|RaceBase_Burrow|TYPE=Base
-###Block
-# Ability Name				Ability
-CATEGORY=Internal|Movement Mode.MOD	ABILITY:Internal|AUTOMATIC|Movement Mode ~ Walk|PREVARGTEQ:Move_Walk,1
-CATEGORY=Internal|Movement Mode.MOD	ABILITY:Internal|AUTOMATIC|Movement Mode ~ Swim|PREVARGTEQ:Move_Swim,1
-CATEGORY=Internal|Movement Mode.MOD	ABILITY:Internal|AUTOMATIC|Movement Mode ~ Fly|PREVARGTEQ:Move_Fly,1
-CATEGORY=Internal|Movement Mode.MOD	ABILITY:Internal|AUTOMATIC|Movement Mode ~ Climb|PREVARGTEQ:Move_Climb,1
-CATEGORY=Internal|Movement Mode.MOD	ABILITY:Internal|AUTOMATIC|Movement Mode ~ Burrow|PREVARGTEQ:Move_Burrow,1
-
-###Block - For setting up correct display of movement. Do not use PREMOVE to detect these movements. It does not detect movement from BONUS tags.
-# Ability Name		Category of Ability	Add to base move
-Movement Mode ~ Walk	CATEGORY:Internal		BONUS:MOVEADD|TYPE.Walk|Move_Walk
-Movement Mode ~ Swim	CATEGORY:Internal		BONUS:MOVEADD|TYPE.Swim|Move_Swim
-Movement Mode ~ Fly	CATEGORY:Internal		BONUS:MOVEADD|TYPE.Fly|Move_Fly
-Movement Mode ~ Climb	CATEGORY:Internal		BONUS:MOVEADD|TYPE.Climb|Move_Climb
-Movement Mode ~ Burrow	CATEGORY:Internal		BONUS:MOVEADD|TYPE.Burrow|Move_Burrow
-
-
-
-#	DEFINE:ReplacesBackgroundFeature|0
-#	BONUS:VAR|ReplacesBackgroundFeature|1|TYPE=Boolean|PREFACT:1,ABILITIES,ReplacesBackgroundFeature=true
\ No newline at end of file
+# CVS $Revision: $ $Author: $ -- Mon Feb 22 22:56:28 2016 -- reformated by PCGen PrettyLST v6.06.00																			
+																			
```

Comment:
I really do not understand the reasoning here... condense all the tab spaces where we have content, but then add filler white space behind everything else, including empty lines.